includes/Admin/Settings.php

```diff
@@ -228,6 +229,114 @@ protected function advanced_options() {
 
 	}
 
+	protected function contact_options() {
```

Comment:
@jjroley Please add docblock comments for each function definition

Reply:
Sorry, I missed this message. Will do that in the future